src/main/java/io/citrine/jcc/search/pif/result/PifMultiSearchResultElement.java

```diff
@@ -0,0 +1,23 @@
+package io.citrine.jcc.search.pif.result;
+
+import io.citrine.jcc.search.core.result.BaseMultiSearchResultElement;
+
+/**
+ * Class to store the information in a single search result as a part of a PIF multi-query.
+ *
+ * @author Kyle Michel
+ */
+public class PifMultiSearchResultElement extends BaseMultiSearchResultElement<PifSearchResult> {
```

Comment:
Are we going to have to do this to hide all generic types?  That seems like a whole lot of code for $\epsilon$ value.  If we wanted to make the usage easier, we could just pick shorter names...